demo/tests/perfs/test_end_point.perf.yml

```diff
@@ -4,6 +4,7 @@ RenderContextSerializer.test_queryset:
 - db: SELECT ... FROM "formidable_item" WHERE "formidable_item"."field_id" IN (...) ORDER BY "formidable_item"."order" ASC
 - db: SELECT ... FROM "formidable_validation" WHERE "formidable_validation"."field_id" IN (...)
 - db: SELECT ... FROM "formidable_default" WHERE "formidable_default"."field_id" IN (...)
+- db: 'SELECT ... FROM "formidable_preset" WHERE "formidable_preset"."form_id" = #'
```

Comment:
it adds a SQL query. I'd be glad to have it retrieved using a "select_related" queryset